File: campus-second-backend/src/main/java/com/campus/entity/Yonghu.java
package com.campus.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

/**
 * 用户表
 */
@Data
@TableName("yonghu")
public class Yonghu {
    @TableId(type = IdType.AUTO)
    private Long id;
    private LocalDateTime addtime;
    private String yonghuzhanghao;
    private String mima;
    private String yonghuxingming;
    private String xingbie;
    private String lianxifangshi;
    private String touxiang;
    private Float money;
}

File: campus-second-frontend/src/api/yonghu.ts
import request from '../utils/request'

export interface YonghuItem {
  id: number
  yonghuzhanghao: string
  mima: string
  yonghuxingming: string
  xingbie: string
  lianxifangshi: string
  touxiang: string
  money: number
}

export const getYonghuList = (params: {
  page?: number
  limit?: number
  keyword?: string
}) => request.get<any>('/api/yonghu/list', { params }).then((res: any) => res.data ?? res)

export const getYonghuDetail = (id: number) =>
  request.get<any>(`/api/yonghu/${id}`).then((res: any) => res.data ?? res)

export const addYonghu = (data: Partial<YonghuItem>) =>
  request.post<any>('/api/yonghu/add', data).then((res: any) => res.data ?? res)

export const updateYonghu = (data: Partial<YonghuItem>) =>
  request.put<any>('/api/yonghu/update', data).then((res: any) => res.data ?? res)

export const deleteYonghu = (id: number) =>
  request.delete<any>(`/api/yonghu/${id}`).then((res: any) => res.data ?? res)

export const recharge = (data: { userid: number; amount: number }) =>
  request.post<any>('/api/yonghu/recharge', data).then((res: any) => res.data ?? res)
